hicommonwealth/commonwealth
# Social login with magic.link

## Link to Issue
Closes: #4009 

## Description of Changes
- Attempts to update #3579 to new login modal.

## Test Plan
- Verify all oauth methods succeed (discord, github, twitter, gmail).
- Verify all 5 auth cases work: new user login, existing user login, new user link, existing user link (merge), and "no-op" link of same user.
